src/components/SQLEditor/SQLEditorSchemaTree.test.tsx

```diff
@@ -1,21 +1,18 @@
 import { render, screen, waitFor, within } from 'test/testUtils';
 import SQLEditorSchemaTree from './SQLEditorSchemaTree';
 import { UserEvent } from '@testing-library/user-event';
-import {
-  schemaTableColumnMock,
-  schemaTablesNonSystemMock,
-} from 'test/__mocks__/schemaTableColumn';
 import { SYSTEM_SCHEMAS } from 'constants/database';
 import _ from 'lodash';
 import { format as formatSQL } from 'sql-formatter';
 import {
   getTablesDDLQueryResult,
   getViewsDDLQueryResult,
 } from 'test/__mocks__/query';
-import { SchemaDescription } from 'contexts';
+import { useSchemaTreeMock } from 'test/__mocks__/useSchemaTreeMock';
+import { SchemaDescription } from 'src/swr/jwt/useSchemaTree';
 
 const getSchemaTableColumns = () => {
-  const schemaTableColumnsParsed = schemaTableColumnMock.rows.map(
+  const schemaTableColumnsParsed = useSchemaTreeMock.rows.map(
```

Comment:
Don't know if it's the same mapping but if it is we could export the `postFetch` method and use it here just to avoid code duplication.